### PR TITLE
FLUT-838891- [Others] UG for the ShowColumnHeaderIconOnHover feature

### PR DESCRIPTION
**Pull request by abineshPalanisamy**
I have included the content and sample code for the following features.
- [Task 836783](https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_workitems/edit/836783): Provide the support to show sort icon when hovering the header cell.

### CODE REVIEW

File: Flutter/datagrid/filtering.md [hunk 1]
```diff
@@ -1325,4 +1325,68 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
\ No newline at end of file
+<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
+
+## Show the column header icons when hovering the header cell
+
+To show a column header icons,such as sort and filter icons when the mouse hovers over a column header in a DataGrid, set the `SfDataGrid.showColumnHeaderIconOnHover` property to true. Note that this feature is only available on web and desktop platforms.
```
Comment (ashok-kuvaraja) [inline Flutter/datagrid/filtering.md:1332]: To display column header icons, such as sort and filter icons, when the mouse hovers over a column header, set the `SfDataGrid.showColumnHeaderIconOnHover` property to true. This feature is available only on web and desktop platforms.

File: Flutter/datagrid/filtering.md [hunk 2]
```diff
@@ -1325,4 +1325,68 @@ Widget build(BuildContext context) {
...
+                child: Text('Salary'))),
+      ],
+    ),
+  );
+}
+  
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="flutter datagrid shows column header icons when hover the header cell" src="images/filtering/flutter-datagrid-show-column-header-icon-on-hover.gif"/>
```
Comment (ashok-kuvaraja) [inline Flutter/datagrid/filtering.md:1392]: flutter datagrid shows column header icons when hovers over the header cell

File: Flutter/datagrid/filtering.md [hunk 3]
```diff
@@ -1325,4 +1325,68 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
\ No newline at end of file
+<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
+
+## Show the column header icons when hovering the header cell
```
Comment (ashok-kuvaraja) [inline Flutter/datagrid/filtering.md:1330]: Move this topic inside the Column Types,

https://help.syncfusion.com/flutter/datagrid/column-types

Comment (ashok-kuvaraja) [inline Flutter/datagrid/filtering.md:1330]: Show the column header icons on hover

File: Flutter/datagrid/filtering.md (hunk 1 above)
Comment (abineshPalanisamy) [reply Flutter/datagrid/filtering.md]: @ashok-kuvaraja - added the content

File: Flutter/datagrid/filtering.md (hunk 2 above)
Comment (abineshPalanisamy) [reply Flutter/datagrid/filtering.md]: @ashok-kuvaraja  -  changed the content

File: Flutter/datagrid/filtering.md (hunk 3 above)
Comment (abineshPalanisamy) [reply Flutter/datagrid/filtering.md]: @ashok-kuvaraja - added to column Types

Comment (abineshPalanisamy) [reply Flutter/datagrid/filtering.md]: @ashok-kuvaraja - Modified the topic name

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine.

**Pull request merged**